core/capabilities/ccip/ccipevm/helpers.go

```diff
@@ -19,15 +20,6 @@ const (
 	evmDestExecDataKey = "destGasAmount"
 )
 
-var (
-	abiUint32               = ABITypeOrPanic("uint32")
-	TokenDestGasOverheadABI = abi.Arguments{
-		{
-			Type: abiUint32,
-		},
-	}
-)
-
 // decodeExtraArgsV1V2 decodes the given EVM Extra Args and extracts the gas limit
 // that was specified.
 func decodeExtraArgsV1V2(extraArgs []byte) (gasLimit *big.Int, err error) {
```

Comment:
I guess we should have a ticket to remove usage of this and always use the extra args codec? @winder

Reply:
I think this was the only usage of it so it should be OK to remove

Reply:
Yep, removed. We already have test coverage for `extraDataDecoder`